doc/observables/FlexibleDecay.rst

```diff
@@ -0,0 +1,4 @@
+======
+FlexibleDecay
+======
+
```

Comment:
Will this be filled? :)

Reply:
We decide we'll fill this while preparing the release.